### DISCUSSION TITLE
HTTP400报错

### DISCUSSION THREAD

**Discussion by dong-dong6** [danny-avila/LibreChat#2500] [Troubleshooting]
### What happened?

![image](https://github.com/danny-avila/LibreChat/assets/73739354/a1ae1600-c89e-4934-9e43-0ce0063209a5)
无法正常调用api，我可以保证api是可用的

### Steps to Reproduce

logs is this

||>User:
你好
||>Assistant:

2024-04-23 11:51:11 error: [MeiliMongooseModel.findOneAndUpdate] Convo not found in MeiliSearch and will index eca1e1e7-e957-4b87-94e6-1e2e436f8b3e Document `eca1e1e7-e957-4b87-94e6-1e2e436f8b3e` not found.

https://oai.itsfurry.com/v1
{
  model: 'gpt-3.5-turbo',
  temperature: 1,
  top_p: 1,
  presence_penalty: 0,
  frequency_penalty: 0,
  stop: [ '||>', '\nUser:', '<|diff_marker|>' ],
  max_tokens: 1024,
  user: '6625e55409c151303acb6838',
  stream: true,
  prompt: '||>Instructions:\n' +
    'You are ChatGPT, a large language model trained by OpenAI. Respond conversationally.\n' +
    'Current date: April 23, 2024\n' +
    '\n' +
    '||>User:\n' +
    '你好\n' +
    '||>Assistant:\n'
}

Response {
  [Symbol(realm)]: null,
  [Symbol(state)]: {
    aborted: false,
    rangeRequested: false,
    timingAllowPassed: true,
    requestIncludesCredentials: true,
    type: 'default',
    status: 400,
    timingInfo: {
      startTime: 115337.10313898325,
      redirectStartTime: 0,
      redirectEndTime: 0,
      postRedirectStartTime: 115337.10313898325,
      finalServiceWorkerStartTime: 0,
      finalNetworkResponseStartTime: 0,
      finalNetworkRequestStartTime: 0,
      endTime: 0,
      encodedBodySize: 416,
      decodedBodySize: 416,
      finalConnectionTimingInfo: null
    },
    cacheState: '',
    statusText: 'Bad Request',
    headersList: HeadersList {
      cookies: null,
      [Symbol(headers map)]: [Map],
      [Symbol(headers map sorted)]: null
    },
    urlList: [ [URL] ],
    body: { stream: undefined }
  },
  [Symbol(headers)]: HeadersList {
    cookies: null,
    [Symbol(headers map)]: Map(11) {
      'date' => [Object],
      'content-type' => [Object],
      'content-length' => [Object],
      'connection' => [Object],
      'x-oneapi-request-id' => [Object],
      'cf-cache-status' => [Object],
      'report-to' => [Object],
      'nel' => [Object],
      'server' => [Object],
      'cf-ray' => [Object],
      'alt-svc' => [Object]
    },
    [Symbol(headers map sorted)]: null
  }
}
Error: Failed to send message. HTTP 400 - {"error":{"message":"The completion operation does not work with the specified model, gpt-35-turbo. Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993. (request id: 20240423195132445413218k2GRn0Od) (request id: 202404231951115447157855O2taOuD)","type":"","param":"","code":"OperationNotSupported"}}
    at onopen (/app/api/app/clients/ChatGPTClient.js:356:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async create (/app/node_modules/@waylaidwanderer/fetch-event-source/lib/cjs/fetch.cjs:56:17) {
  status: 400,
  json: {
    error: {
      message: 'The completion operation does not work with the specified model, gpt-35-turbo. Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993. (request id: 20240423195132445413218k2GRn0Od) (request id: 202404231951115447157855O2taOuD)',
      type: '',
      param: '',
      code: 'OperationNotSupported'
    }
  }
}
2024-04-23 11:51:32 error: [handleAbortError] AI response error; aborting request: Failed to send message. HTTP 400 - {"error":{"message":"The completion operation does not work with the specified model, gpt-35-turbo. Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993. (request id: 20240423195132445413218k2GRn0Od) (request id: 202404231951115447157855O2taOuD)","type":"","param":"","code":"OperationNotSupported"}}

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Some issue with your reverse proxy: 
![image](https://github.com/danny-avila/LibreChat/assets/110412045/89eb6f8c-f7ce-4dfe-b10e-14232d274a8c)

**Reply by dong-dong6** [reply]
I encountered this error after adding v1 to my proxy address. If v1 is not added, the model cannot be found

**Comment by jcdiv47** [comment]
> Error: Failed to send message. HTTP 400 - {"error":{"message":"The completion operation does not work with the specified model, gpt-35-turbo. Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993.

@dong-dong6 Just want to make sure you are aware of this, is there a model named `gpt-35-turbo` on your server? Or is it a typo where it might be `gpt-3.5-turbo`?

**Reply by dong-dong6** [reply]
我不知道为什么出现这个情况，但是我在更改gid和uid为0之后，将不再报错
